i3pystatus/cpu_freq.py

```diff
@@ -0,0 +1,54 @@
+"""
+The module gathers information by default from `/proc/cpuinfo` about the current cpu frequency
+"""
+# coding=utf-8
+from i3pystatus import IntervalModule
+
+
+class CpuFreq(IntervalModule):
+    """
+    class uses by default `/proc/cpuinfo` to determine the current cpu frequency
+
+    .. rubric:: Available formatters
+
+    * `{avg}` - mean from all cores in MHz `4.3f`
+    * `{avgg}` - mean from all cores in GHz `1.2f`
+    * `{corex}` - frequency of a selected core in MHz `4.3f`
+    * `{corexg}` - frequesncy of a selscted core in GHz `1.2f`
```

Comment:
It's not very clear from the description that `x` is just a placeholder here, maybe rephrase it into `{coreX} - frequency of core X in ...` or so